script.js

```diff
@@ -85,15 +117,49 @@ function createEpisodeListItem(episode) {
   return episodeListItem;
 }
 
+function fillShowList() {
```

Comment:
A more conventional name for this function could be `populateShowSelector`

Reply:
done